VRDR.CLI/Program.cs

```diff
@@ -762,15 +762,15 @@ record = coding.DeathRecord;
                         var d = submission.DeathRecord;
                         IJEMortality ije1 = new IJEMortality(d, false);
                         // Loop over every property (these are the fields); Order by priority
-                        List<PropertyInfo> properties = typeof(IJEMortality).GetProperties().ToList().OrderBy(p => p.GetCustomAttribute<IJEField>().Priority).ToList();
+                        List<PropertyInfo> properties = typeof(IJEMortality).GetProperties().ToList().OrderBy(p => p.GetCustomAttribute<IJEField>().Location).ToList();

```

Comment:
Would this mean the priority values that we use for the death record fields get ignored? Does that matter?

Reply:
The priority is used in Canary (I think), but for dumping the IJE values we really want the dump to be in the same order as the fields in the IJE record, and that is Location.